layouts/project/single.html

```diff
@@ -2,29 +2,26 @@
 	<div class="margin-bottom-30">
 		{{ .Content }}
 	</div>
-	{{ if gt (len .Params.eclipse_projects) 0 }}
-	<h2 class="margin-bottom-40">Eclipse projects</h2>
-	<div class="row">
-	{{ range .Params.eclipse_projects }}
-		{{range first 1 (where $.Site.Data.eclipsefdn_projects "project_id" .) }}
-		<div class="col-sm-12 margin-bottom-30 match-height-item-by-row flex-column project-item">
-		  <div class="padding-bottom-20 text-center background-white flex-grow">
-		    <div >
-		      <h3>
-		        <a href="{{ .url }}">
-			      <img class="img img-responsive margin-auto padding-bottom-20 " src="{{ .logo }}" title="{{ .name }} logo"/>
-				  {{ .name }}
-			    </a>
-			  </h3>
+
+	{{ if .Params.referenced_eclipse_projects }}
+		{{ if gt (len .Params.referenced_eclipse_projects) 0 }}
+			<h2 class="margin-top-30 margin-bottom-20">Referenced Eclipse projects</h2>
+			<div class="row">
+			{{ range .Params.referenced_eclipse_projects }}
+			    {{range first 1 (where $.Site.Data.eclipsefdn_projects "project_id" .) }}
+			    <div class="col-md-4 background-white margin-bottom-20 match-height-item vertical-align margin-right-20">
```

Comment:
This should not be creating a new well for every logo. We should be displaying this content much like we are doing  for the Eclipse Project Contributions section

Reply:
By well do you mean the white background?

Reply:
Sort of. I mean the individual white boxes. These logos should be in one larger box that spans the width of the available area, and be an inline list (like the other logos later in the page).